spatialpandas/io/parquet.py

```diff
@@ -16,6 +16,7 @@
 from dask.utils import natural_sort_key
 from pandas.io.parquet import to_parquet as pd_to_parquet
 from pyarrow import parquet as pq
+from pyarrow.parquet import ParquetDataset, read_metadata
```

Comment:
It would be nice if there was only one `pyarrow` import here rather than two separate `import`s. Could you choose one or the other?